tests/playwright_statistics_test.py

```
@@ -0,0 +1,269 @@
+"""
+Author: rahn
+Datum: 16.08.2025
+Version: 1.0
+Beschreibung: Playwright Tests für Statistics Tab - Model Combination Issue
+ÄNDERUNG 16.08.2025: Tests für Statistics-Tab Modell-Kombination vs. Einzelmodell-Problem
+"""
+
+import asyncio
+import json
+import time
```

Ruff (0.12.2)

`time` imported but unused

Remove unused import: `time`

(F401)